docs/developing-hardware/flags.rst

```diff
@@ -25,7 +25,7 @@ Clash Compiler Flags
 
   - ``DebugFinal`` to show expressions after they have been completely
     normalized
-
+  - ``DebugCount`` to count how often each transformation is applied
```

Comment:
Could you document this in `docs/developing-hardware/flags.rst`?

Reply:
Also, the "ordered debug levels" is really starting to work against us. We should rework it to a set of debug options.

Reply:
Didn't I just do that?

Reply:
Damnit I'm dumb, sorry.